.travis/opensuse.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -xe
-${DOCKER_EXEC} zypper -n ar -f -G http://download.opensuse.org/repositories/devel:/tools/openSUSE_13.2/devel:tools.repo
+OS_VERSION=$(${DOCKER_EXEC} cat /etc/os-release | awk -F[=\"] '/PRETTY_NAME=/ {print $3}')
```

Comment:
@hirenvadalia : I agree with @arungrover . Instead of pretty_name we should make use of NAME and VERSION. 13.2 may not come back but we don't know what will be PRETTY_NAME for next release.

Man page of os-release  
      NAME=
           A string identifying the operating system, without a version component, and suitable for presentation to the user. If not
           set, defaults to "NAME=Linux". Example: "NAME=Fedora" or "NAME="Debian GNU/Linux"".

       VERSION=
           A string identifying the operating system version, excluding any OS name information, possibly including a release code name,
           and suitable for presentation to the user. This field is optional. Example: "VERSION=17" or "VERSION="17 (Beefy Miracle)"".

      PRETTY_NAME=
           A pretty operating system name in a format suitable for presentation to the user. May or may not contain a release code name
           or OS version of some kind, as suitable. If not set, defaults to "PRETTY_NAME="Linux"". Example: "PRETTY_NAME="Fedora 17
           (Beefy Miracle)"".

PRETT_NAME -> May or may not contain a release code name or OS version of some kind, as suitable. I think it is better to depend on NAME and VERSION.